blockchain/chain/src/store/chain_store.rs

```diff
@@ -217,6 +188,40 @@ where
     }
 }
 
+/// Returns a Tuple of bls messages of type UnsignedMessage and secp messages
+/// of type SignedMessage
+pub fn messages<DB>(
```

Comment:
Optional change but messages seems like a bad name for this and could be improved to indicate what it actually does